Gemfile.lock

```diff
@@ -217,15 +217,17 @@ GEM
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.3.6)
-    mini_portile2 (2.8.4)
     minima (2.5.1)
       jekyll (>= 3.5, < 5.0)
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
     minitest (5.20.0)
     mutex_m (0.1.2)
-    nokogiri (1.15.4)
-      mini_portile2 (~> 2.8.2)
+    nokogiri (1.15.4-x64-mingw-ucrt)
```

Comment:
maybe we should add this to git exclude? I'm not sure if updating this file is necessary on a commit.

Reply:
I see. My docker image tends to update the gems. 

By the way, I think GitHub hosing uses this file.

Reply:
ah ok, nevermind then